### PR TITLE
[Magiclysm] Shapeshifter Alkahest spell

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "Added new spell to shapeshifters attunement"

#### Purpose of change
-

#### Describe the solution
The big AoE ground controll spell with small dot

#### Describe alternatives you've considered
-
#### Testing
Works as intended

#### Additional context
Due the lack of possibility the spell can also kill NPCs, so i added info about it in the description
also ill be happy if someone edit the current description

### CODE REVIEW

**Comment by KorGgenT** [comment]
what exactly is alkahest?

File: data/mods/Magiclysm/Spells/attunements/Shapeshifter.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "type": "SPELL",
+    "id": "shapeshifter_alkahest",
+    "name": "Alkahest",
+    "description": "Release the mana-saturated liquid out your ley lines and cover everything with bioactive solvent.  Sadly you are the only person that have resistance to his effect, so its better to be alone when you cast it.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Shapeshifter.json:6]: this description needs a little bit of grammatical work.

File: data/mods/Magiclysm/Spells/attunements/Shapeshifter.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "effect": "attack",
+    "shape": "cone",
+    "damage_type": "acid",
+    "max_level": 25,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Shapeshifter.json:12]: attunement spells are supposed to be balanced around max level 35

**Comment by GuardianDll** [comment]
> what exactly is alkahest?

https://en.wikipedia.org/wiki/Alkahest

File: data/mods/Magiclysm/Spells/attunements/Shapeshifter.json (hunk 2 above)
Comment (GuardianDll) [reply data/mods/Magiclysm/Spells/attunements/Shapeshifter.json:12]: whoops, my bad

File: data/mods/Magiclysm/Spells/attunements/Shapeshifter.json (hunk 1 above)
Comment (KorGgenT) [reply data/mods/Magiclysm/Spells/attunements/Shapeshifter.json:6]: maybe something like 
"Release a spray of mana-saturated bioactive solvent. Your ley lines give you a resistance to this acid due to your mana signature, but others beware!"
I know it's a slightly different message so take the suggestion as you will

Comment (GuardianDll) [reply data/mods/Magiclysm/Spells/attunements/Shapeshifter.json:6]: no problem, in original message i said it must be rewrited, cause im not the best in description stuff

**Review by KorGgenT — APPROVED** [review]

**Pull request merged**